tests/new_api_test.cpp

```diff
@@ -1345,6 +1350,252 @@ void test_remove_corrupted(const ioremap::elliptics::newapi::session &session) {
 	BOOST_REQUIRE_EQUAL(result.status(), 0);
 }
 
+void test_bulk_read(const ioremap::elliptics::newapi::session &session) {
```

Comment:
It still doesn't check bulk_read for reading one key from several groups living on the same node. Different groups should have different values (json & data) for this key.